tests/integration/records/single_db/test_records_load.py

```diff
@@ -10,12 +10,15 @@
 
 class RecordsLoadIntegrationTest(BaseRecordsIntegrationTest):
     def load_and_verify(self, format_type, variant, hints={}, broken=False, sourcefn=None):
-        if self.engine.name == 'bigquery' and variant == 'csv':
+        redshift_with_no_bucket = self.engine.name == 'redshift' and not self.has_scratch_bucket()
+        # TODO: See if the issue I'm seeing in tests is related too
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker